styles/components/_scoreTable.scss

```diff
@@ -0,0 +1,36 @@
+.scores {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: $border-radius;
+  background-color: $color-primary;
+
+  &__header {
+    padding: 1.25rem;
```

Comment:
please organize the properties by:
// Positioning /
// Display & Box Model /
// Color /
// Text /
// Other /

Reply:
will do

Reply:
done